components/event-bus/api/publish/v2/api.go

```diff
@@ -0,0 +1,53 @@
+package v2
+
+const (
+	// FieldData value
+	FieldData = "data"
+	// FieldEventID value
+	FieldEventID = "id"
+	// FieldEventTime value
+	FieldEventTime = "time"
+	// FieldEventType value
+	FieldEventType = "type"
+	// FieldSpecVersion value
+	FieldSpecVersion = "specversion"
+	// FieldEventTypeVersion value
+	FieldEventTypeVersion = "eventtypeversion"
```

Comment:
The `eventtypeversion` is hard to read, please make it more readable.

Reply:
it follows the specs convention